trials/showImage.js

```diff
@@ -16,49 +16,47 @@ const { jitter50 } = require("../lib/utils");
  * @param {boolean} config.USE_ELECTRON - USE_ELECTRON flag
  * @param {boolean} config.USE_MTURK - USE_MTURK flag
  * @param {Object} options
- * @param {string} options.responseType - This tells jsPsych which plugin file to use to run the trial.
- * @param {number} options.duration - The trial duration in milliseconds.
- * @param {string} options.image - The path of the image file to be displayed.
- * @param {number} options.imageHeight - Set the height of the image in pixels. If left null (no value specified), then the image will display at its natural height.
- * @param {number} options.imageWidth - Set the width of the image in pixels. If left null (no value specified), then the image will display at its natural width.
- * @param {boolean} options.responseEndsTrial - True if the trial ends on response,false if the trial waits for the duration, by default false value.
- * @param {number} options.taskCode - Task code to be saved into data log and for pdSpotEncode, which by default is null and is passed when config has USE_PHOTODIODE set true.
- * @param {number} options.numBlinks - Number of times the pulse needs to be repeated for photodiode box, when USE_PHOTODIODE is set true. If not set, by default is 1.
+ * @param {number} options.duration - trial duration in milliseconds. (default: 1000)
+ * @param {string} options.image - The path of the image file to be displayed. (default: a blue dot image)
```

Comment:
will this default work outside of the context of this repo? might make more sense to have image required and move it out of the options block so `showImage(config, image, options)`